mobile.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/mobile.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="main-nav">
+            <div class="status-bar">
+                <div class="time-style">
+                    <p class="time">9:14</p>
+                </div>
```

Comment:
- [ ] Please ensure that you include a logo on the left side of the page and ensure it is wrapped in `<a>` tag.
- [ ] I'm also unable to see the hamburger icon which should be on the right side of the page. Please ensure you add it and wrap it inside the `<nav>` tag. You can use[ fontawsome](https://fontawesome.com/) to get the hamburger icon.